server/src/resolvers/Mutation.js

```diff
@@ -0,0 +1,58 @@
+module.exports.Mutation = {
+  signInWithGoogle: async (_, {googleId}, {dataSources}) => {
+    const user = await dataSources.mainAPI.findOneUser({googleId});
+    if (user === null) {
+      return {
+        success: false,
+        message: 'User not found. You have to sign up first.',
```

Comment:
These strings are user facing. The ideal way to organize such strings is put these user-facing assets (strings, icons, graphics, etc) in a separate folder and import them here. For now, let's at least collect these at the top of the file as a block of constants, and leave a TODO to move them to appropriate resource folder later.

Reply:
What is the better solution? I think we can return enum type(error code) from the back-end server and the front-end can handle this enum type to show the error message to the user. or we can pass the error message directly from the server.

Reply:
Yeah be should return the status only and FE can render different strings depending on the status code. Not super urgent so we can do this later.